src/main/java/utils/parser/DeckKeywordParser.java

```diff
@@ -28,8 +31,8 @@ private static Options buildDeleteOptions() {
         Options options = new Options();
         options.addRequiredOption("d", "deck", true, "deck name");
         // extra stuff down here
-        //        options.addOption("c", "card", true, "card name");
```

Comment:
could you remove this comment too

Reply:
Comment has been removed in d812969b5cf5c1267130c8931110096a535d65f0